frontend/src/components/App/App.js

```diff
@@ -57,15 +57,17 @@ function App() {
             lineName: item.lineName,
             timeToStation: item.timeToStation
           }));
-        const abridgedData = abridgeData(filteredData);
-        // line below was used before quantisedData was added to state for data visualiser
-        const quantisedData = quantiseData(abridgedData).sort((a, b) => a.timeToStation - b.timeToStation);
-
-        setVisualData(quantisedData); // added for data visualiser
-        // setQuantisedData(quantiseData(abridgedData));
-        // localStorage.setItem('quantisedData', JSON.stringify(quantisedData, null, 2)); // FOR DATA COLLECTION ONLY
-        console.log("in fetchData")
-        console.log(quantisedData);
+          const sortedData = filteredData.sort((a, b) => a.timeToStation - b.timeToStation);
+          // STEP 1: remove unnecessary text, whitespace, and apostrophes.
+          const abridgedData = abridgeData(sortedData);
+          // STEP 2: separate data into individual lines.
+          const separatedData = separateDataIntoLines(abridgedData);
```

Comment:
personally I think there should just be one function call for the separateDataIntoLines, fudgeData, and quantiseData , ideally called spreadData? As this describes what we are trying to do with the data pretty well and 'fudge' makes it sound like we are doing something dodgy to it! lol .. I think these functions should be called from within one enclosing function and the complexity hidden away from app.js